cincinnati/src/plugins/catalog.rs

```diff
@@ -10,6 +10,9 @@ use self::cincinnati::plugins::BoxedPlugin;
 use super::internal::arch_filter::ArchFilterPlugin;
 use super::internal::channel_filter::ChannelFilterPlugin;
 use super::internal::cincinnati_graph_fetch::CincinnatiGraphFetchPlugin;
+use super::internal::dkrv2_openshift_secondary_metadata_scraper::{
```

Comment:
Whats up with the name `dkrv2`, does it have a full form?

Reply:
It's a reference to the [Docker V2](https://docs.docker.com/registry/spec/api/) HTTP API that is implemented by dkregistry-rs.